docker/etc/s6/gitea/default_env

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Define the environment variables
+export DATA_PATH=${DATA_PATH:-"/data/gitea"}
```

Comment:
Yes I didn't want to cluter the dockerfile but still wanted to have a dedicated file detailing all the env variable possible to use and their default. But it could be in the Dockerfile.

Reply:
I added a comment to the Dockerfile to indicate to go see this file.

Reply:
I agree that the various config options seem to clutter the Dockerfile.

Removing the `GITEA_CUSTOM` environment variable breaks the ad-hoc commands.

```
# docker run -d --name gitea_no_env gitea:sapk
# docker exec -it gitea_no_env gitea dump
2019/04/17 13:14:08 ...s/setting/setting.go:501:NewContext() [W] Custom config '/usr/local/bin/custom/conf/app.ini' not found, ignore this if you're running first time
<snip>
```

Reply:
Why `DATA_PATH`? `APP_DATA_PATH` is already a setting in `app.ini` we should keep name same for same purpose.